crates/apps_lib/src/client/rpc.rs

```diff
@@ -874,6 +882,7 @@ pub async fn query_protocol_parameters(
         epochs_per_year,
         masp_epoch_multiplier,
         masp_fee_payment_gas_limit,
+        masp_nam_shielding_fee: masp_nam_shielding_fee.to_string_precise(),
```

Comment:
Would it be possible to also return the price for the other supported tokens?

Reply:
In theory, but at genesis I think only supporting nam is fine